Patches/[O21] Outer Rim Galaxies/Mandalore/Patch_OuterRim_Mandalore_Weapons.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>[O21] Outer Rim Galaxies [DevBuild]</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>
+		
+			<li Class="PatchOperationConditional">
+				<xpath>Defs/ThingDef[defName="O21_OR_AmbanSniperRifle"]</xpath>
+				<match Class="PatchOperationSequence">
+					<operations>
+						
+					  <li Class="PatchOperationReplace">
+						<xpath>Defs/ThingDef[
+						defName="O21_OR_WESTAR34Blaster" or
+						defName="O21_OR_WESTAR35Blaster"
+						]/tools</xpath>
+						<value>
+						  <tools>
+							<li Class="CombatExtended.ToolCE">
+							  <label>grip</label>
+							  <capacities>
+								<li>Blunt</li>
+							  </capacities>
+							  <power>2</power>
+							  <cooldownTime>1.54</cooldownTime>
+							  <chanceFactor>1.5</chanceFactor>
+							  <armorPenetrationBlunt>0.555</armorPenetrationBlunt>
+							  <linkedBodyPartsGroup>Grip</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+							  <label>muzzle</label>
+							  <capacities>
+								<li>Poke</li>
+							  </capacities>
+							  <power>2</power>
+							  <cooldownTime>1.54</cooldownTime>
+							  <armorPenetrationBlunt>0.555</armorPenetrationBlunt>
+							  <linkedBodyPartsGroup>Muzzle</linkedBodyPartsGroup>
+							</li>
+						  </tools>
+						</value>
+					  </li>
+
+				  <li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[		
+					defName="Amban_Rifle"			
```

Comment:
This was named to `O21_OR_AmbanSniperRifle,` swap out references to it.

Reply:
Done